modules/javafx.swing/src/main/java/javafx/embed/swing/JFXPanel.java

```diff
@@ -446,13 +446,11 @@ protected void processMouseEvent(MouseEvent e) {
             (e.getButton() == MouseEvent.BUTTON1)) {
             if (isFocusable() && !hasFocus()) {
                 requestFocus();
-                // this focus request event goes to eventqueue and will be
-                // asynchronously handled so MOUSE_PRESSED event will not be
-                // honoured by FX immediately due to lack of focus in fx
-                // component. Fire the same MOUSE_PRESSED event after
-                // requestFocus() so that 2nd mouse press will be honoured
-                // since now fx have focus
-                jfxPanelIOP.postEvent(this, e);
+                // This fixes JDK-8087914 without causing JDK-8200224
```

Comment:
We actually do not clutter source code with bugids, it will be good if it can be removed with proper comments maybe something like "extra simulated mouse pressed event is removed by making the JavaFX scene focused"

Reply:
In general, we no longer reference bug IDs in source code, so I agree with the recommendation to reword the comment.

@FlorianKirmaier - can you reword as suggested? Once you are done, you can integrate it (using the `/integrate` command), and I will sponsor it. There will likely be a delay due to the US Thanksgiving holiday.

Reply:
Done